<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionsaveAs"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuVIew">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionzoomIn"/>
    <addaction name="actionzoomOut"/>
    <addaction name="actionPrev"/>
    <addaction name="actionNext"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionblur"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVIew"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="separator"/>
   <addaction name="actionzoomIn"/>
   <addaction name="actionzoomOut"/>
   <addaction name="actionPrev"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionblur"/>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionsaveAs">
   <property name="text">
    <string>saveAs</string>
   </property>
   <property name="toolTip">
    <string>saveAs</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
   <property name="toolTip">
    <string>exit</string>
   </property>
  </action>
  <action name="actionzoomIn">
   <property name="text">
    <string>zoomIn</string>
   </property>
  </action>
  <action name="actionzoomOut">
   <property name="text">
    <string>zoomOut</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionblur">
   <property name="text">
    <string>blur</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
